docs/pages/404.tsx

```diff
@@ -30,13 +30,13 @@ function ConstructionIllustration() {
   );
 }
 
-// Modifying this code may have security implications
-// See.. https://github.com/keystonejs/keystone/pull/6411#issuecomment-906085389
+// modifying this code may have security implications
+//   see https://github.com/keystonejs/keystone/pull/6411#issuecomment-906085389
 const v5PathList = ['/tutorials', '/guides', '/keystonejs', '/api', '/discussions'];
 
 export default function NotFoundPage() {
   const { asPath } = useRouter();
-  const tryV5Link = asPath.startsWith('/') && v5PathList.some(i => asPath.startsWith(i));
+  const tryV5Link = v5PathList.some(x => asPath.startsWith(x));
```

Comment:
This would be the preferred approach that we would want for the AdminUI, to be honest